app/assets/javascripts/characterisation.js

```diff
@@ -1,44 +0,0 @@
-(function ($, undefined) {
```

Comment:
The SetCharacterisationDescriptors task was only used in two defunct pipelines, which have been unused since operations moved the old Illumina-C pipelines into Limber. They got disabled a few months back, so now clearing out the code. There is a *lot* of code.